Fixing theme related issues (#2765)
wso2/docs-apim: en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

# Overriding Developer Portal Theme

There are mainly 2 ways to customize and override the developer portal theme. 

> 1. Theming using defaultTheme.js

Reviewer: ```suggestion
1. Theming using `defaultTheme.js`
```

2. Overriding React components

> In this section, we will cover the first technique which is the easiest way to change the appearance of Developer portal.

Reviewer: Shall we use the word `method` instead of the word `technique` 😃  just a suggestion @shaniR WDYT ?

```suggestion
In this section, we will cover the first method which is the easiest way to change the appearance of Developer portal.
```

> For the second technique, please refer [Advanced Customization]({{base_path}}/develop/customizations/advanced-ui-customization/) section.

Reviewer: ```suggestion
For the second method, please refer [Advanced Customization]({{base_path}}/develop/customizations/advanced-ui-customization/) section.
```

  }
}
```

> The following documents describe some of the frequent theming tasks that you can achieve by customizing the defaultTheme.js.

Reviewer: ```suggestion
The following documents describe some of the most commonly used customizations, that you can do by changing the `defaultTheme.js`
```
